components/signup/signup_email/signup_email.tsx

```diff
@@ -122,13 +138,13 @@ export default class SignupEmail extends React.PureComponent {
         }
     }
 
-    handleSignupSuccess = (user, data) => {
+    handleSignupSuccess = (user: UserSignUpInfo, data: UserProfile) => {
```

Comment:
I think the type of `User` will also be `UserProfile`, and since we're asserting the type on the object we pass in, I think that should be accepted automatically. It's weird that we're passing basically duplicate data as both parameters though.

Reply:
Just as a heads up, Typescript ended up wanting me to explicitly type the user parameter as UserProfile, so on my fix, both parameters will have the same twice.